BASE/Test/Microsoft.ApplicationInsights.Test/ApplicationInsightsTypes/Generated/Data_types.cs

```diff
@@ -45,7 +45,7 @@ public Data()
 
         protected Data(string fullName, string name)
         {
-            baseData = global::Bond.GenericFactory.Create<TDomain>();
+            baseData = default; // global::Bond.GenericFactory.Create<TDomain>();
```

Comment:
Is this functionality wise the same?

Reply:
doesn't affect any product code, only our unit tests.

The description for `Bond.GenericFactory` says it would initialize to the default value.
https://github.com/microsoft/bond/blob/master/cs/src/core/GenericFactory.cs

Reply:
What does "the default value" mean - is it the same with C#'s "default" or these two defaults are actually different level of concepts?